File: easy/fizzBuzz/fizzBuzz.py
# Makes Python 2.x and 3.x play nice.
try: input = raw_input
except NameError: pass

def fizzBuzz(n):
    str = ""
    for i in range(1,n+1):
        temp = ""
        if i % 3 == 0:
            temp += "Fizz"
        if i % 5 == 0:
            temp += "Buzz"
        if len(temp) == 0:
            temp = i
        str += "{}\n".format(temp)
    return str

print(fizzBuzz(int(input("Enter an integer greater than 0: "))))

File: easy/fizzBuzz/fizzBuzz.java
import java.util.Scanner;

public class fizzBuzz {
	public static void main(String[]args) {
		Scanner scan = new Scanner(System.in);
		System.out.print("Enter an integer greater than 0: ");
		System.out.println(fizzBuzz(scan.nextInt()));
	}

	public static String fizzBuzz(int n) {
		String output = "";
		for (int i = 1; i <= n; i++) {
			String temp = "";
			if (i % 3 == 0) {
				temp += "Fizz";
			}
			if (i % 5 == 0) {
				temp += "Buzz";
			}
			if (temp.length() == 0) {
				temp += i;
			}
			output += temp+"\n";
		}
		return output;
	}
}